configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,40 @@
+{
+    "activeSlotsCoeff": 0.05,
+    "systemStart": "2017-09-23T09:44:51Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 764824073,
+    "networkId": "Mainnet",
+    "epochLength": 216000,
+    "securityParam": 2160,
+    "slotLength": 1,
+    "slotsPerKESPeriod": 216000,
+    "maxKESEvolutions": 60,
+    "protocolParams": {
+        "a0": 0,
+        "decentralisationParam": 1,
+        "eMax": 0,
+        "extraEntropy": {
+            "tag": "NeutralNonce"
+        },
+        "keyDeposit": 0,
+        "maxBlockBodySize": 2097152,
+        "maxBlockHeaderSize": 8192,
+        "maxTxSize": 2048,
+        "minFeeA": 0,
+        "minFeeB": 0,
+        "minUTxOValue": 1,
+        "minPoolCost": 100,
+        "nOpt": 100,
+        "poolDeposit": 0,
+        "protocolVersion": {
+            "major": 2,
+            "minor": 0
+        },
```

Comment:
And this doesn't get used when in composite mode. Would also be nice to be optional, using e.g. 0,0 as default.